composer.json

```diff
@@ -1,22 +1,22 @@
 {
-    "name": "pimcore/elastic-search",
+    "name": "byng/pimcore-elasticsearch-plugin",
+    "description": "Elasticsearch Pimcore plugin",
+    "version": "2.0.0",
     "type": "pimcore-plugin",
-    "license": "BSD-3-Clause",
-    "description": "Elastic Search Pimcore plugin",
-    "keywords": ["pimcore", "elastic search", "plugin"],
-    "homepage": "http://byng.co",
+    "license": "MIT",
+    "keywords": [ "pimcore", "elasticsearch", "plugin" ],
+    "homepage": "https://github.com/byng-systems/pimcore-elastic-search-plugin",
     "authors": [
         {
             "name": "Byng Systems",
```

Comment:
I really think we should start putting down specific names here (using the commit log as a reference).  Thoughts?